spec/services/plans/prepare_for_override_service_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Plans::PrepareForOverrideService, type: :service do
+  subject(:prepare_service) { described_class.new(organization, plan.code) }
+
+  let(:membership) { create(:membership) }
+  let(:organization) { membership.organization }
+  let(:plan) { create(:plan, organization: organization) }
+  let(:standard_charge) { create(:standard_charge) }
+
+  let(:params) do
+    {
+      trial_period: 1,
+      amount_cents: 200,
+      amount_currency: 'EUR',
+      charges: [
+        {
+          id: standard_charge.id,
+          charge_model: 'standard',
+          properties: {
+            amount: '0.22',
+          },
+        }
+      ]
+    }
+  end
+
+  let(:expected_params_without_code) do
+    {
+      name: plan.name,
+      description: plan.description,
+      bill_charges_monthly: plan.bill_charges_monthly,
+      interval: plan.interval,
+      pay_in_advance: plan.pay_in_advance,
+      overridden_plan_id: plan.id,
+      organization_id: organization.id,
+      trial_period: 1,
+      amount_cents: 200,
+      amount_currency: 'EUR',
+      charges: [
+        {
+          billable_metric_id: standard_charge.billable_metric_id,
+          charge_model: 'standard',
+          properties: {
+            amount: '0.22',
+          },
+        }
```

Comment:
Style/TrailingCommaInArrayLiteral: Put a comma after the last item of a multiline array. (https://rubystyle.guide#no-trailing-array-commas)